<ns0:entry xmlns:ns0="http://www.w3.org/2005/Atom" xmlns:ns1="http://schemas.google.com/g/2005" xmlns:ns2="http://schemas.google.com/projecthosting/issues/2009" ns1:etag="W/&quot;DkEAQ347eCl7ImA9WhZTE04.&quot;">
		<ns0:id>http://code.google.com/feeds/issues/p/nativeclient/issues/1319/comments/full/6</ns0:id><ns0:author>
			<ns0:name>mseaborn@chromium.org</ns0:name><ns0:uri>/u/mseaborn@chromium.org/</ns0:uri></ns0:author><ns0:content type="html">See also issue 896 ("Plugin: onfail not called if process fails to start").

There is RunOnfailHandler() in the plugin for eval'ing an "onfail" attribute but it appears not to be used in the PPAPI case.</ns0:content><ns0:updated>2011-03-17T04:04:02.000Z</ns0:updated><ns0:published>2011-03-17T04:04:02.000Z</ns0:published><ns2:updates /><ns0:title>Comment 6 by mseaborn@chromium.org</ns0:title><ns0:link href="http://code.google.com/p/nativeclient/issues/detail?id=1319#c6" rel="alternate" type="text/html" /><ns0:link href="https://code.google.com/feeds/issues/p/nativeclient/issues/1319/comments/full/6" rel="self" type="application/atom+xml" /></ns0:entry>